### DISCUSSION TITLE
The RocketFuel Theme in released for Oqtane 5

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#3491] [Show and tell]
The theme has been upgraded to 5.3.2 of Bootstrap
The Theme can be downloaded from the Market Place / Oqtane Registry.

**Comment by thabaum** [comment]
@leigh-pointer Took your theme out for a test drive today.

![image](https://github.com/oqtane/oqtane.framework/assets/13577556/cca1b25c-8a3a-4412-9ef6-8e9f41a295ec)

clicking on Admin Dashboard results in a blank page with theme loaded.

![image](https://github.com/oqtane/oqtane.framework/assets/13577556/ea592274-5740-4294-b4b3-11db252e3a28)

Thought I would share!  Maybe a Z axis css is pushing it behind since if I load right to the site/admin page and then hit that button the modal is visible.

**Reply by leigh-pointer** [reply]
@thabaum thanks for the update.  Indeed its not firing on all cylinders! I will release a fix shortly.

**Reply by leigh-pointer** [reply]
@thabaum 5.0.2 ready for down load.  This version comes with an Anchor # feature on the Container Title bar - when in admin mode.

![image](https://github.com/oqtane/oqtane.framework/assets/9447612/1a886374-300a-4d81-bb1c-3e941a4b1699)

**Reply by thabaum** [reply]
@leigh-pointer awesome man, I will take it for a test drive soon!
